src/GitHubCodeReviewDashboard/GitHubCodeReviewDashboard/GitHubCodeReviewDashboard.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <UserSecretsId>c8280495-0f86-4936-ace1-48033adf44da</UserSecretsId>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Humanizer" Version="2.8.11" />
```

Comment:
is humanizer speciest to non-humans?

Reply:
The PR explicitly excludes PRs from dotnet-bot, so yes we are ignoring non-humans.